tools/packaging/common/istio.service

```diff
@@ -1,12 +1,13 @@
 [Unit]
 Description=istio-sidecar: The Istio sidecar
 Documentation=http://istio.io/
+StartLimitIntervalSec=0
 
 [Service]
 ExecStart=/usr/local/bin/istio-start.sh
+ExecStartPre=+/usr/bin/install -d -o istio-proxy -m 0755 /var/run/secrets
```

Comment:
Does this work universally? I tried it locally and ran into issues:
```
$ /usr/bin/install -d -o howardjohn -m 0755 /var/run/secrets
/usr/bin/install: cannot change owner and permissions of ‘/var/run/secrets’: No such file or directory
```

Note our CI  does test VMs, but it invokes istio-start.sh directly so we wouldn't have coverage of this

Reply:
Did you run as root? The `+` indicates that it's run with elevated privileges. Tested it on debian 11 and my ubuntu laptop.

> Note our CI does test VMs, but it invokes istio-start.sh directly so we wouldn't have coverage of this

I could add a check to the istio-start script to verify the directory exists. Imo envoy shouldn't be able to start (and run into an error) if the directory isn't there in the first place.